posts/2022/xarray-groupby-vs-geocat-climatology.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<p>contentental -&gt; continental</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Hmm you can't have sea surface temperatures over the continental US...</p>

Reply:
<p>Yes this was a holdover from when I was using a dataset that focused on the coastal US. I forgot to change the text when I changed the dataset.</p>

Reply:
<p>I'm still a little confused about the values over land (with the updated plots). When I tried to look up this <a href="https://www.psl.noaa.gov/data/gridded/data.noaa.oisst.v2.html" target="_blank">dataset online</a> it appears that values over land should be masked. @dcherian any insight on this particular ocean dataset?</p>

Reply:
Yes you're right Katie. This dataset is only sea surface temperatures. For simplicity in this example, I didn't mask the land. I didn't think masking was necessary since this example is looking at the Eastern Pacific and the purpose of the demonstration is showing how to compute the seasonal averages. Do you think I should add a mask really quick?

Reply:
Sure I think a mask might help with any confusion, thanks!